tests/Unit/Aggregate/AggregateChangedTest.php

```diff
@@ -10,29 +10,31 @@
 use Patchlevel\EventSourcing\Aggregate\AggregateChangeRecordedAlready;
 use Patchlevel\EventSourcing\Tests\Unit\Fixture\Email;
 use Patchlevel\EventSourcing\Tests\Unit\Fixture\ProfileCreated;
+use Patchlevel\EventSourcing\Tests\Unit\Fixture\ProfileCreatedWithCustomRecordedOn;
 use Patchlevel\EventSourcing\Tests\Unit\Fixture\ProfileId;
 use PHPUnit\Framework\TestCase;
 
 use const PHP_VERSION_ID;
 
+/** @covers \Patchlevel\EventSourcing\Aggregate\AggregateChanged */
```

Comment:
can we use attributes here? :) 

```php
PHPUnit\Framework\Attributes\CoversClass(string $className)
```

https://github.com/sebastianbergmann/phpunit/issues/4502

Reply:
nvm, phpunit v10 is not released yet :D